Libraries/LibC/sys/select.cpp

```diff
@@ -28,13 +28,26 @@
 #include <errno.h>
 #include <stdio.h>
 #include <sys/select.h>
+#include <sys/time.h>
 
 extern "C" {
 
-int select(int nfds, fd_set* readfds, fd_set* writefds, fd_set* exceptfds, struct timeval* timeout)
+int select(int nfds, fd_set* readfds, fd_set* writefds, fd_set* exceptfds, timeval* timeout_tv)
 {
-    Syscall::SC_select_params params { nfds, readfds, writefds, exceptfds, timeout };
+    timespec* timeout_ts = nullptr;
+    timespec timeout;
+    if (timeout_tv) {
+      timeout_ts = &timeout;
+      TIMEVAL_TO_TIMESPEC(timeout_tv, timeout_ts);
+    }
```

Comment:
Please use `clang-format`.

Reply:
I tried! I think the problem was that I passed `--diff` to `git clang-format` which made it print a diff instead of apply it.

This seems to work well to clang-format all commits on a branch, without needing any user input:

    git rebase -x 'git clang-format HEAD^ && git commit -a --amend --no-edit' master